docs/dev/Installation_and_setup.md

```diff
@@ -0,0 +1,152 @@
+## Overview
+
+The Open Database Connectivity (ODBC) driver lets you integrate Open Distro for Elasticsearch with your favorite Business Intelligence (BI) applications.
+
+The driver is currently supported for MacOS and Windows platform. The latest version of driver is compatible with ODBC **3.51** specification.
+
+### Supported Versions
+
+  | Operating System  | Version |
+  | ------------- |-------------|
+  |  Windows    |  Windows 10 |
+  |  MacOS    |  Catalina 10.15.4, Mojave 10.14.6 |
+
+## Concepts
+
+ |      |   |
+ | ------------- |-------------|
+ |  **DSN**    | A Data Source Name is used to store driver information in the system. By storing the information in the system, the information does not need to be specified each time the driver connects. |
+ |  **.tdc** file    | The TDC file contains configuration information that Tableau applies to any connection matching the database vendor name and driver name defined in the file. This configuration allows customers to fine-tune parts of their ODBC data connection, turn on/off certain features not supported by data source|
+
+  
+## Driver Installation
+
+The driver can be obtained by downloading the bundled distribution installer from [here](https://opendistro.github.io/for-elasticsearch/downloads.html) or by building from source.
```

Comment:
You can link to the build instructions (<root>/BUILD_INSTRUCTIONS.md), which has instructions for building the driver from source.